pkg/controller/controller.go

```diff
@@ -14,11 +14,13 @@ import (
 	messagingcmd "github.com/hyperledger/aries-framework-go/pkg/controller/command/messaging"
 	routercmd "github.com/hyperledger/aries-framework-go/pkg/controller/command/route"
 	vdricmd "github.com/hyperledger/aries-framework-go/pkg/controller/command/vdri"
+	"github.com/hyperledger/aries-framework-go/pkg/controller/command/verifiable"
 	"github.com/hyperledger/aries-framework-go/pkg/controller/rest"
 	didexchangerest "github.com/hyperledger/aries-framework-go/pkg/controller/rest/didexchange"
 	messagingrest "github.com/hyperledger/aries-framework-go/pkg/controller/rest/messaging"
 	"github.com/hyperledger/aries-framework-go/pkg/controller/rest/route"
 	vdrirest "github.com/hyperledger/aries-framework-go/pkg/controller/rest/vdri"
+	verifiableest "github.com/hyperledger/aries-framework-go/pkg/controller/rest/verifiable"
```

Comment:
whats est?

Reply:
supposed to be rest :) will update it.

Reply:
[Updated](https://github.com/hyperledger/aries-framework-go/compare/f80627c99b434a3b201a73595a5abc864dd72949..c8e18af1df281112d620a62c9034670f5c2ed84c#diff-3c862eb54a8e0e161b534e0c67e5379eR23).